volumes/miovision/sql/function/function-exclude_denylist_from_intersection_movements.sql

```diff
@@ -0,0 +1,39 @@
+CREATE OR REPLACE FUNCTION miovision_api.exclude_denylist_from_intersection_movements()
+RETURNS trigger
+LANGUAGE plpgsql
+AS $BODY$
+
+DECLARE row_in_im numeric = (
+    SELECT COUNT(*)
+    FROM miovision_api.intersection_movements_denylist AS im
```

Comment:
is it possible to make the table name a param?

Reply:
Done!